rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/10_synonyms_put.yml

```diff
@@ -13,6 +13,8 @@
               id: "test-id"
 
   - match: { result: "created" }
+  - match: { reload_analyzers_details._shards.total: 0 }
```

Comment:
I'm not sure we should include reload analyzer details on create. WDYT?

Reply:
I thought about this, but than I thought it is better for response to look the same and not different depending on conditions. What do you think?

Reply:
Synchronized with @carlosdelest offline, and we decided to keep the response as is for 8.9, and may revisit for GA.